src/index.js

```diff
@@ -39,7 +42,9 @@ export default class Swup {
 			plugins: [],
 			skipPopStateHandling: function(event) {
 				return !(event.state && event.state.source === 'swup');
-			}
+			},
+			fragmentTargetAttr: 'data-swup-to-fragment',
+			fragmentContainerAttr: 'data-swup-fragment-container'
```

Comment:
I would think one of these would end up being `data-swup-to-fragment`, probably the second one. I think we should avoid the word `container` when it comes to fragments, to prevent any confusion with the classic containers. 

I'm also thinking the options could probably go in one of the ways:  
1. Be a completely editable selector like the containers are, but then I get that we need to be able to save a value in those attributes as well. 
2. Be a fixed name.

Do you see some value in it being editable with the way the fragments API is set up? It seems like it just adds a bit of complexity in the setup instead of helping anything. Is there some situation where the attribute name would be better to be custom?

Reply:
I put them into the options for convenience while building the feature; I agree these probably shouldn't be editable and could get removed from the options. The advantage here is that the options object is basically available everywhere and we don't need to figure out how to pass the selector around.